boards/arm/numaker_pfm_m467/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
empty cmake? can you also add board after adding SoC support to preserve bisectability?

Reply:
OK, it's fine to remove this empty CMakeLists.txt .
About preserve bisectability, is it meaning to exchange the commit sequence of "board" and "soc", to let soc as 1st commit ?

Reply:
yes, board should be the latest.